src/test/java/carleton/sysc4907/ui/view/ElementLibraryPanelTest.java

```diff
@@ -78,6 +77,7 @@ private void start(Stage stage) throws IOException {
             DependencyInjector injector = new DependencyInjector();
 
             Manager mockManager = Mockito.mock(Manager.class);
+            lenient().when(mockManager.isHost()).thenReturn(true);
```

Comment:
May be helpful to have a comment about this - it is important but not obvious what it is for.